src/CSET/operators/read.py

```diff
@@ -534,6 +535,13 @@ def _fix_um_lightning(cube: iris.cube.Cube):
         pass
 
 
+def _fix_model_level_name(cube: iris.cube.Cube):
+    """To fix model level varname for consistency to allow merging."""
```

Comment:
```suggestion
    """Fix LFRic model level varname for consistency to allow merging."""
```
Note the model this is needed for.